db/primary_migrate/20240604173515_add_sponsor_id_to_in_person_enrollment.rb

```diff
@@ -0,0 +1,5 @@
+class AddSponsorIdToInPersonEnrollment < ActiveRecord::Migration[7.1]
+  def change
+    add_column :in_person_enrollments, :sponsor_id, :string
```

Comment:
I'm not familiar with how this column is expected to be used, but would we expect to query such that we'd want to create an index on the column?

Reply:
Good point! I actually can't think of a requirement to query this field today.... 🤔 I guess their might be a time where we'd like to see how many folks went through the Enhanced IPP flow, but I'm assuming that would be retrieved from the analytics. Let me think on this a bit more. Thanks!

Reply:
I'd probably lean towards no as well.